modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -479,6 +479,22 @@ definition {
 			value1 = "\RETURN");
 	}
 
+	@summary = "Verify if the Error Message show"
+	macro verifyErrorMessege {
+		if (isSet(AssertElementPresent)) {
+			AssertElementNotPresent(
+				key_errorMessage = ${errorMessage},
+				locator1 = "PRMMDFRequest#ERROR_MESSAGE");
+		}
+	}
+
+	@summary = "Verify if the Error Message not show"
```

Comment:
Hi @pedroamarall, I think that your macro summary could be 'Verify that the error message is not displayed', what do you think?

Reply:
Done, thanks!